pandas/compat/__init__.py

```diff
@@ -71,7 +71,30 @@ def bytes_to_str(b, encoding=None):
         return b.decode(encoding or 'utf-8')
 
     def signature(f):
-        return list(inspect.signature(f).parameters.keys())
+        from collections import namedtuple
+        sig = inspect.signature(f)
```

Comment:
can you add a refernce to that django issue here (and comment that we copied directly)